proto/dydxprotocol/subaccounts/query.proto

```diff
@@ -29,7 +29,15 @@ service Query {
       QueryGetWithdrawalAndTransfersBlockedInfoRequest)
       returns (QueryGetWithdrawalAndTransfersBlockedInfoResponse) {
     option (google.api.http).get =
-        "/dydxprotocol/subaccounts/withdrawals_and_transfers_blocked_info";
+        "/dydxprotocol/subaccounts/withdrawals_and_transfers_blocked_info/"
+        "{perpetual_id}";
```

Comment:
Is this change supposed to be part of this change?

Reply:
Yes, I noticed I didn't update the URL in a previous PR so rolled it up into this one. Going to update the description.